src/cmd.js

```diff
@@ -144,6 +145,32 @@ function handleSequenceList(
   return batch ? result : result.then(() => []);
 }
 
+function handleScheduleCmd(cmd, dispatch, getState, loopConfig) {
+  let setIntervalOrTimeout = cmd.isRepeating ? setInterval : setTimeout;
+  const handle = setIntervalOrTimeout(() => {
+    const cmdPromise = executeCmd(
+      cmd.nestedCmd,
+      dispatch,
+      getState,
+      loopConfig
+    );
+    if (cmdPromise) {
+      cmdPromise.then(actions => {
+        if (!actions.length) {
+          return;
+        }
+        return Promise.all(actions.map(dispatch));
```

Comment:
there's no need to return here. nothing is looking  at the result of the promise. just say `actions.map(dispatch)`.

Reply:
actually, map and forEach pass extra params, which we should not do. so i would say `actions.forEach(action => dispatch(action))`

Reply:
Yep. Thanks, I missed this one.

This brings up a more important issue though. By using `dispatch()` internally (instead of returning promises) `Cmd.setTimeout` and `Cmd.setInterval` will have the same gotcha as `Cmd.list`. When wrapping these commands in `Cmd.map`, the tagger is not called and sub-actions are dispatched directly to the root reducer. I would like to fix that gotcha before we release this new feature.

Reply:
let's merge my other pr for that issue first, and then you can rebase this pr and address the issue. You had one comment I needed clarification on, but once you're good with it I'll merge the PR

Reply:
my pr is merged

Reply:
This PR is using `wrappedDispatch` now.